resources/definitions/Geeetech_A30.def.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
This file name needs to be the same as the ID, so it should be `geeetech_a30.def.json`